index.html

```diff
@@ -84,7 +84,7 @@
   </footer>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.10/handlebars.min.js"></script>
   <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
-  <script src="scripts/rawData.json" type="text/javascript"></script>
+  <!-- You were calling the JSON file as a script here, no need -->
```

Comment:
For future reference, when making contributions like this I usually delete the line without introducing a comment, then add a line note in the PR to explain the deletion. Or call out the change specifically in the commit message.